packager/mpd/base/mpd_utils.h

```diff
@@ -47,7 +47,8 @@ std::string GetCodecs(const MediaInfo& media_info);
 std::string GetBaseCodec(const MediaInfo& media_info);
 
 // Returns a key made from the characteristics that separate AdaptationSets.
-std::string GetAdaptationSetKey(const MediaInfo& media_info);
+std::string GetAdaptationSetKey(const MediaInfo& media_info,
+           bool no_codec = false);
```

Comment:
We don't normally allow default parameters, which could hide problems. Please do not use default parameter here.